distribution/archives/build.gradle

```diff
@@ -330,12 +330,6 @@ configure(subprojects.findAll { it.name == 'integ-test-zip' }) {
 
   group = "org.elasticsearch.distribution.integ-test-zip"
 
-  restResources {
-    restTests {
-      includeCore '*'
-    }
-  }
-
   integTest {
```

Comment:
I just realized that moving these tests out of this project and into :rest-api-spec has the side effect of changing 
from: running these tests if anything changes (due to the dependsOn assemble) 
to: only running the tests if the any of the actual tests change.   

Do we want to keep the behavior of always running the core rest tests ?